pyproject.toml

```diff
@@ -7,7 +7,10 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9, <3.12"
+structlog = "^23.2.0"
+fastapi = "^0.108.0"
```

Comment:
get rid of fastapi and rdlib please, a logger shouldn't need app specific packages.